tests/unit/test_util.py

```diff
@@ -88,9 +88,11 @@ def test_get_parameters_filepath(self):
             path = os.path.join(package_dir, tempfile_obj.name)
             assert pybamm.get_parameters_filepath(tempfile_obj.name) == path
 
-    @pytest.mark.skipif(pybamm.have_jax(), reason="The JAX solver is not installed")
-    def test_is_jax_compatible(self):
-        assert True
+
+@pytest.mark.skipif(not pybamm.have_jax(), reason="JAX is not installed")
+def test_is_jax_compatible():
+    compatible = pybamm.is_jax_compatible()
+    assert compatible
```

Comment:
I still think this is unneeded, but if it is being put in, then it needs to be cleaned up
```suggestion
    @pytest.mark.skipif(not pybamm.have_jax(), reason="JAX is not installed")
    def test_is_jax_compatible():
        assert pybamm.is_jax_compatible()
```